sdk/attestation/tests.yml

```diff
@@ -3,4 +3,16 @@ trigger: none
 extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
   parameters:
-    ServiceDirectory: attestation
\ No newline at end of file
+    ServiceDirectory: attestation
+    Location: westus
+    EnvVars:
+      ISOLATED_ATTESTATION_URL: $(ISOLATED_ATTESTATION_URL)
+      AAD_ATTESTATION_URL: $(AAD_ATTESTATION_URL)
+      serializedPolicySigningKey0: $(serializedPolicySigningKey0)
```

Comment:
Where do you expect these environment variables to come from?

Reply:
They come from the test-resources.env file created by new-testresources.ps1. THe test pipeline failed because it didn't find them, I assumed that meant that the tooling didn't import them from the test-resources.env file.